agent/agent-tooling/src/main/java/com/microsoft/applicationinsights/agent/internal/wasbootstrap/configuration/Configuration.java

```diff
@@ -418,4 +420,15 @@ public void validate() throws FriendlyException {
     // transient so that Moshi will ignore when binding from json
     public transient long lastModifiedTime;
 
+    public static class ProfilerConfiguration {
+        public int configPollPeriod = -1;
+        public int periodicRecordingDuration = -1;
+        public int periodicRecordingInterval = -1;
+        public String serviceProfilerFrontEndPoint = null;
```

Comment:
do we expect this to be used by end users, or is it only for testing?

Reply:
if you are referring to the serviceProfilerFrontEndPoint then I would doubt a user would ever need to set it in practice, it does help in test with routing data to test endpoints